sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ServiceBusSessionProcessorOptions.cs

```diff
@@ -153,6 +153,11 @@ public int MaxConcurrentCallsPerSession
         /// </summary>
         public IList<string> SessionIds { get; } = new List<string>();
 
+        /// <summary>
+        /// Number of messages to receive.
+        /// </summary>
+        public int BatchSize { get; set; } = 1;
```

Comment:
should we check this is greater than 0 when set?

Reply:
updated!